queue/src/fiskaltrust.Middleware.Localization.QueueIT/Services/SigningDeviceIT.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using fiskaltrust.ifPOS.v1.it;
+using Newtonsoft.Json;
+using fiskaltrust.Middleware.Contracts.Interfaces;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT.Services
+{
+    public class SigningDeviceIT : ISigningDevice
+    {
+        private readonly IITSSCD _client;
+        private readonly ILogger _logger;
+
+
+        public SigningDeviceIT(IITSSCDProvider itIsscdProvider, ILogger<SigningDeviceIT> logger)
+        {
+            _client = itIsscdProvider.Instance;
+            _logger = logger;
+        }
+
+        public async Task<bool> IsSigningDeviceAvailable()
+        {
+            try
+            {
+                var deviceInfo = await _client.GetDeviceInfoAsync().ConfigureAwait(false);
+                _logger.LogInformation(JsonConvert.SerializeObject(deviceInfo));
```

Comment:
```suggestion
                _logger.LogDebugJsonConvert.SerializeObject(deviceInfo));
```
I think as thats potentially printed a lot debug verbosity is enough 🤔